### DISCUSSION TITLE
如何用python订阅直播间？

### DISCUSSION THREAD

**Discussion by logos000** [xfgryujk/blivechat#217] [提问]
我看了一下通过websocket用房间号订阅应该是这样的：
    async def subscribe(self, room_id):
        await self.websocket.send(json.dumps({
            "cmd": 1,
            "data": {"roomId": room_id}
        }))
        print(f"Subscribed to room {room_id}")
       但是我没找到怎么用身份码订阅，然后我现在想用身份码应该怎么做呢

**Comment by xfgryujk** [comment]
你想偷我的接口？

**Reply by logos000** [reply]
不好意思啊，不行的话我就不搞了
